src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFilesIntegrationTest.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.*;
+
+import static htsjdk.samtools.ValidationStringency.STRICT;
+
+public class CreateVariantIngestFilesIntegrationTest extends CommandLineProgramTest {
+
+    private static final String input_vcf_file = "NA12878.haplotypeCalls.reblocked.chr20.100k.vcf.gz";
```

Comment:
@kcibul is it better practice to move all of these inside the test? I'd thought eventually we may have other tests that would use them, but for now there's just the one.
OR if these should stay here, should I also move the "expected" file names out here as well? it's not consistent right now and I don't have any real rationale for their being defined one place or another.

Reply:
yeah -- either way seems reasonable to me.  FWIW I think I would put them inside the test for now... but that's just personal preference

Reply:
great, moved them inside the test - can always refactor if/when we add more tests if they use the same files